test/util/server.go

```diff
@@ -119,7 +119,13 @@ func DefaultMasterOptions() (*configapi.MasterConfig, error) {
 		return nil, err
 	}
 
-	return startOptions.MasterArgs.BuildSerializeableMasterConfig()
+	masterConfig, err := startOptions.MasterArgs.BuildSerializeableMasterConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	masterConfig.ServiceAccountConfig.LimitSecretReferences = true
```

Comment:
interesting... I like it, but a comment about why we're not using a default would be good to keep this from getting removed by someone